.gitignore

```diff
@@ -23,7 +23,7 @@ terraform/*.tf.json
 
 # Vagrant VM artifacts
 ubuntu*.log
-/vagrant
+/.vagrant
```

Comment:
This is in my .gitignore_global file and it works well. Maybe it doesn't matter to have leading `/`.
```
.vagrant/
```

Reply:
the leading `/` does matter.. otherwise we could have another situation like this:

https://github.com/airbnb/streamalert/pull/980/files